src/main/java/org/jenkinsci/plugins/GithubAccessTokenProperty.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2017, CloudBees, Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.jenkinsci.plugins;
+
+import hudson.model.UserProperty;
+import hudson.util.Secret;
+
+import javax.annotation.Nonnull;
+
+/**
+ * Remembers the access token used to connect to the Github server
+ *
+ * @since TODO
+ */
+public class GithubAccessTokenProperty extends UserProperty {
```

Comment:
I suspect you need to create at least an empty `config.jelly`. Did you check what the web UI shows when you `/me/configure` with this patch?

While [`summary` is optional](https://github.com/jenkinsci/jenkins/blob/7d29d4df37ae5602fed41430e8a67eedbe76889f/core/src/main/resources/hudson/model/User/index.jelly#L39), currently it seems [`config` is not](https://github.com/jenkinsci/jenkins/blob/7d29d4df37ae5602fed41430e8a67eedbe76889f/core/src/main/resources/hudson/model/User/configure.jelly#L52).

Reply:
Thanks to you comment, I tested that page and yes it throws an exception. I just had to implement the method "isEnabled" and return false in the descriptor like that was done for ApiTokenProperty. For the moment there is no need to have visual representation of the GitHub access token (it is not visible in the GH page so not configurable manually)